app/views/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1>Posts</h1>
+<%= @post.each do |post| %>
+<div class="post">
+  <h2><a href="post_path"</a>@post</h2>
```

Comment:
Your link isn't written out properly. Need to put some content between the `<a></a>` tags. Also need to close the opening tag.